lib/ansiblelint/__main__.py

```diff
@@ -127,7 +128,7 @@ def main():
             options.verbosity = options.verbosity + config['verbosity']
 
         if 'exclude_paths' in config:
-            options.exclude_paths = options.exclude_paths + config['exclude_paths']
+            options.exclude_paths.extend(config['exclude_paths'])
```

Comment:
Since you are already refactoring it, let's just do
```suggestion
        options.exclude_paths.extend(
            config.get('exclude_paths', []),
        )
```